ARK Server Manager/OpenRCON.xaml.cs

```diff
@@ -48,8 +52,32 @@ public OpenRCON()
             this.DataContext = this;
         }
 
+        public static int GetMaxPlayers()
+        {
+            
+            JsonSerializer serializer = new JsonSerializer();
+            using (var client = new WebClient())
```

Comment:
The only reason I did this that way is because UpdateArkDataAsync() was not returning the player's image. I put a breakpoint on line 116 of PlayerInfo.cs and it would never get to it. I mean, there was 156 images it had to download,but I'm not sure it would take over 5 minutes to get all of them.